cpp/tests/io/parquet_test.cpp

```diff
@@ -716,6 +716,59 @@ TEST_F(ParquetWriterTest, Strings)
   cudf::test::expect_metadata_equal(expected_metadata, result.metadata);
 }
 
+TEST_F(ParquetWriterTest, StringsAsBinary)
+{
+  std::vector<const char*> unicode_strings{
+    "Monday", "Wȅdnȅsday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+  std::vector<const char*> ascii_strings{
+    "Monday", "Wednesday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+
+  column_wrapper<cudf::string_view> col0{ascii_strings.begin(), ascii_strings.end()};
+  column_wrapper<cudf::string_view> col1{unicode_strings.begin(), unicode_strings.end()};
+  column_wrapper<cudf::string_view> col2{ascii_strings.begin(), ascii_strings.end()};
+  cudf::test::lists_column_wrapper<int8_t> col3{{'M', 'o', 'n', 'd', 'a', 'y'},
```

Comment:
:] A missed opportunity here to quote from The Godfather: ["Maunday, Toosday, Thursday, Wednesday..."](https://www.youtube.com/watch?v=5-4ZZj023uA)